packages/mui-base/src/Select/Select.test.tsx

```diff
@@ -1253,20 +1220,19 @@ describe('<Select />', () => {
 
     const select = getByRole('combobox');
     act(() => {
-      select.focus();
-      fireEvent.mouseDown(select); // opens and highlights '1'
+      select.click(); // opens the listbox
     });
 
     // React renders twice in strict mode, so we expect twice the number of spy calls
-    expect(renderOption1Spy.callCount).to.equal(2);
+    expect(renderOption1Spy.callCount).to.equal(2); // '1' as focusVisible becomes true
 
-    fireEvent.keyDown(select, { key: 'ArrowDown' }); // highlights '2'
-    expect(renderOption1Spy.callCount).to.equal(4); // '1' rerenders as it loses highlight
-    expect(renderOption2Spy.callCount).to.equal(2); // '2' rerenders as it receives highlight
+    await userEvent.keyboard('{ArrowDown}'); // highlights '2'
+    expect(renderOption1Spy.callCount).to.equal(6); // '1' rerenders as it loses highlight
```

Comment:
For these, the addition of `useButton` into `useOption` adds 1 extra rerender to each interaction (which becomes 2 in strict mode).

Reply:
Yeah, that's unfortunate. It's caused by the focus-visible polyfill that's built in the useButton. I hope we'll be able to remove it soon.